Zilon.Core/Zilon.Emulation.Common/AutoplayEngineBase.cs

```diff
@@ -46,7 +46,7 @@ public async Task StartAsync(IPerson startPerson, IServiceProvider serviceProvid
             {
                 try
                 {
-                    gameLoop.Update();
+                    var actorsWithoutTasks = gameLoop.UpdateAsync();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'actorsWithoutTasks'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)